Client/TabManagement/TabMigrationUtility.swift

```diff
@@ -39,15 +46,32 @@ class DefaultTabMigrationUtility: TabMigrationUtility {
         return shouldRunMigration
     }
 
-    func runMigration(savedTabs: [LegacySavedTab]) async -> WindowData {
-        logger.log("Begin tab migration with legacy tab count \(savedTabs.count)",
+    func getLegacyTabs() -> [LegacySavedTab] {
+        guard let tabData = legacyTabDataRetriever.getTabData() else {
+            return []
+        }
+
+        do {
+            let unarchiver = try NSKeyedUnarchiver(forReadingFrom: tabData)
+            guard let tabs = unarchiver.decodeDecodable([LegacySavedTab].self, forKey: tabsKey) else {
+                // let message = "\(unarchiver.error?.localizedDescription ?? "Couldn't decode from tabsKey")"
```

Comment:
Can we delete this comment line?

Reply:
Oops, yes I will remove it in the next PR to avoid rerunning BR.